app/controllers/hyrax/dashboard/nest_collections_controller.rb

```diff
@@ -18,6 +21,19 @@ def create_within
         end
       end
 
+      def process_nesting 
+        byebug
+         @parent_id = params[:parent_id]
+         @child_id = params[:child_id]
+         
+         parent_coll = Collection.find params[:parent_id]
+         child_coll = Collection.find params[:child_id]
+         parent_coll.add_member_objects ( params[:child_id] )
```

Comment:
You will want to use this instead of `add_member_objects`...
```
child_col1.member_of_collections << parent_col1
```
This sets up the inverse relationship `<child><pcdm:isMemberOf><parent>`